blog/_posts/2018-04-11-scala-3.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: blog-page
```

Comment:
There's also a Jekyll build warning:

    Build Warning: Layout 'blog-page' requested in blog/_posts/2018-04-11-scala-3.md does not exist.

You might need to check the front matter of you blog post is similar to other recent blog posts.

Reply:
Thanks! There were indeed some differences.